dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/MetadataResourceHelper.java

```diff
@@ -24,6 +41,8 @@ public static String createWorkflowURL(Workflow workflow) {
             return baseUrl + "/workflows/" + workflow.getWorkflowPath();
         } else if (workflow instanceof Service) {
             return baseUrl + "/services/" + workflow.getWorkflowPath();
+        } else if (workflow instanceof AppTool) {
+            return baseUrl + "/apptools/" + workflow.getWorkflowPath();
```

Comment:
I don't think this is right; I believe it should be `"/containers/"` (but please verify). An example url for an app tool on qa is `https://qa.dockstore.org/containers/github.com/common-workflow-library/bio-cwl-tools/bandage-info:release?tab=info`, with `containers`, so I think it means it has to be.